src/librustc_metadata/schema.rs

```diff
@@ -182,6 +182,11 @@ pub struct CrateRoot<'tcx> {
 
     pub entries_index: LazySeq<index::Index<'tcx>>,
 
+    /// The data for any proc macros declared by this crate
+    /// These are stored in the same order as the proc macros
+    /// are stored under their dylib symbol
+    pub proc_macro_data: Option<LazySeq<ProcMacroData>>,
```

Comment:
Not sure you need this, you can rely entirely on attributes, I think?

Reply:
See my comment above.

Reply:
Okay so we can defer this.